themes/bootstrap3/templates/myresearch/mylist.phtml

```diff
@@ -71,6 +71,9 @@
   <?php if ($list && !empty($list->description)): ?>
     <p><?=$this->escapeHtml($list->description)?></p>
   <?php endif; ?>
+  <?php if (!empty($listTags)): ?>
+    <div><?=$this->transEsc('Your Tags')?>: <?=implode(', ', array_map([$this, 'escapeHtml'], $listTags))?></div>
```

Comment:
I wonder if this label should simply be "Tags" since in public mode, the tags don't belong to the viewing user.